scripts/test-iaccount-execute.py

```
@@ -0,0 +1,497 @@
+            return False
+
+    elif selector == ESTIMATE_REVERT_AT_MAX_SELECTOR:
+        # Decode EstimateCallGasRevertAtMax(bytes)
+        decoded = decode(
+            ["bytes"],
+            bytes.fromhex(error_data[10:] if error_data.startswith("0x") else error_data[8:]),
+        )
+        revert_data = decoded[0]
+        if not expect_success:
+            print(f"  PASS {description}")
+            print(f"       EstimateCallGasRevertAtMax (expected failure)")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)